aiohttp/web_protocol.py

```diff
@@ -308,10 +308,16 @@ def connection_made(self, transport: asyncio.BaseTransport) -> None:
         if self._tcp_keepalive:
             tcp_keepalive(real_transport)
 
-        self._task_handler = self._loop.create_task(self.start())
         assert self._manager is not None
         self._manager.connection_made(self, real_transport)
 
+        loop = self._loop
```

Comment:
Moved to after `connection_made` since it will begin to execute immediately and the semantics previously may have expected the connection to always be there before the task began to execute.